recipes/nose2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: nose2
+  version: "0.7.4"
+
+source:
+  fn: nose2-0.7.4.tar.gz
+  url: https://files.pythonhosted.org/packages/93/46/a389a65237d0520bb4a98fc174fdf6568ad9dcc79b9c1d1f30afc6776031/nose2-0.7.4.tar.gz
```

Comment:
Is this on PyPI?

Reply:
Yes, I used "conda-skeleton pypi" to build the recipe, and just cleaned few things afterwards.